### ISSUE TITLE
Medallia change request- Appointments Pre-checkin Request to remove A11y intercept survey on urls

### ISSUE THREAD

**Issue by aubreyarcangel** [department-of-veterans-affairs/va.gov-team#89182]
## Issue Description

# [URLs to Exclude](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/Medalia/urls-to-exclude-from-intercept-survey.md) from the Medallia Intercept Survey

These URLs should be excluded from the list of URLs from which the Medallia a11y Intercept Survey will trigger.
> **_NOTE:_**  **Any of the URLs below may contain a query string used for language, modal, or error flags**

> **_NOTE:_**  **Some of the URLs below, as noted, will have an appended UUID (appended to the URL, not a query string)**

## eCheck-in
- /health-care/appointment-check-in/verify
- /health-care/appointment-check-in/loading-appointments
- /health-care/appointment-check-in/appointments
- /health-care/appointment-check-in/arrived
- /health-care/appointment-check-in/contact-information
- /health-care/appointment-check-in/emergency-contact
- /health-care/appointment-check-in/next-of-kin
- /health-care/appointment-check-in/travel-pay
- /health-care/appointment-check-in/travel-mileage
- /health-care/appointment-check-in/travel-vehicle
- /health-care/appointment-check-in/travel-address
- /health-care/appointment-check-in/travel-review
- /health-care/appointment-check-in/travel-agreement
- /health-care/appointment-check-in/details
- /health-care/appointment-check-in/error
- This URL will have an appended (not query string) UUID
  - /health-care/appointment-check-in/appointment-details

## Pre-Check-in 
- /health-care/appointment-pre-check-in/verify
- /health-care/appointment-pre-check-in/appointments
- /health-care/appointment-pre-check-in/introduction
- /health-care/appointment-pre-check-in/contact-information
- /health-care/appointment-pre-check-in/emergency-contact
- /health-care/appointment-pre-check-in/next-of-kin
- /health-care/appointment-pre-check-in/error
- This URL will have an appended (not query string) UUID
  - /health-care/appointment-pre-check-in/appointment-details

## OH Travel
- /my-health/appointment-travel-claim/verify
- /my-health/appointment-travel-claim/travel-pay
- /my-health/appointment-travel-claim/travel-mileage
- /my-health/appointment-travel-claim/travel-vehicle
- /my-health/appointment-travel-claim/travel-address
- /my-health/appointment-travel-claim/travel-review
- /my-health/appointment-travel-claim/travel-agreement

---
## Tasks
- [ ] Coordinate with VSignals team to perform request
- [ ] QA each url

## Acceptance Criteria
- [ ] A11y intercept survey does not show on urls listed
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by aubreyarcangel** [comment]
Submitted to VEO intake portal. Ticket ID B4F67FF

**Comment by ATMiddleton** [comment]
Duplicate of #88729
